Makefile

```diff
@@ -21,10 +21,10 @@ dist: ; $(info creating dist directory...)
 .PHONY: prereqs prereqs-force
 
 prereqs: ; $(info installing dev tooling...) 
-	@source ./hack/install-devtools.sh
+	./hack/install-devtools.sh
```

Comment:
The use of `source` was needed in order to affect the environment variables (specifically, ensure $GOPATH/bin is in the PATH).
Without `source`, the command is invoked in a child shell and any environment change is reverted upon exit.
So the change, as is, would break that.

According to the SO question you linked, alternatives might be to add `SHELL=/bin/bash` to the Makefile (`source` will be found then since it's a bash built-in) or use dot (`.`) as the POSIX way of sourcing (never tried this in makefile, quick check in a shell shows it works):
```make
   @. ./hack/install-devtools.sh
```